src/cli/git-scaffold/.gitignore

```
@@ -0,0 +1,5 @@
+dist
+test/node_modules
+test/dist
+*/**/target
```

Pretty sure this can be simplified to:

```suggestion
dist/
node_modules/
target/
```